google/cloud/storage/examples/storage_object_acl_samples.cc

```diff
@@ -105,6 +105,25 @@ void DeleteObjectAcl(gcs::Client client, int& argc, char* argv[]) {
   (std::move(client), bucket_name, object_name, entity);
 }
 
+void GetObjectAcl(gcs::Client client, int& argc, char* argv[]) {
+  if (argc < 4) {
+    throw Usage{"get-object-acl <bucket-name> <object-name> <entity>"};
+  }
+  auto bucket_name = ConsumeArg(argc, argv);
+  auto object_name = ConsumeArg(argc, argv);
+  auto entity = ConsumeArg(argc, argv);
+  //! [get object acl] [START storage_get_file_acl]
+  [](google::cloud::storage::Client client, std::string bucket_name,
+     std::string object_name, std::string entity) {
+    google::cloud::storage::ObjectAccessControl acl =
```

Comment:
Line is a bit long. I'm starting to think that samples could benefit from a namespace alias. It helps to show the type so I'd recommend keeping it.

Reply:
Ack. We could start all samples with:

```C++
namespace gcs = google::cloud::storage;
```

Reply:
+1, that sounds good to me.